### PR TITLE
Update Go to 1.15.5

### PR DESCRIPTION
**Pull request by tklauser**

### CODE REVIEW

**Comment by tklauser** [comment]
test-me-please

File: images/hubble-proto/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM docker.io/library/golang:1.15.3-alpine3.12 as builder
```
Comment (kaworu) [inline images/hubble-proto/Dockerfile:1]: Did you change this one manually? `images/hubble-proto` should be added to _images/scripts/update-golang-image.sh_ right?

**Review by kaworu — APPROVED** [review]

File: images/hubble-proto/Dockerfile (hunk 1 above)
Comment (tklauser) [reply images/hubble-proto/Dockerfile:1]: Yeah, so far I've been doing this manually (but apparently forgot on the last update). I agree, this should be part of `images/scripts/update-golang-image.sh`.

Comment (kaworu) [reply images/hubble-proto/Dockerfile:1]: I'll work on a patch once this PR is merged to avoid conflict.

**Review by aanm — APPROVED** [review]

**Review by christarazi — APPROVED** [review]

**Pull request merged**